protos/notify.proto

```diff
@@ -0,0 +1,83 @@
+syntax = "proto3";
+
+package protos;
+
+option go_package = "github.com/streamdal/snitch-protos/build/go/protos";
+
+enum NotificationType {
+  NOTIFICATION_TYPE_UNSET = 0;
+  NOTIFICATION_TYPE_SLACK = 1;
+  NOTIFICATION_TYPE_EMAIL = 2;
+  NOTIFICATION_TYPE_PAGERDUTY = 3;
+}
+
+message NotificationConfig {
+  optional string id = 1;
+
+  optional string name = 2;
+
+  optional NotificationType type = 3;
+
+  oneof config {
+    NotificationSlack slack = 1000;
+    NotificationEmail email = 1001;
+    NotificationPagerDuty pagerduty = 1002;
+  }
+}
+
+message NotificationSlack {
+  optional string bot_token = 1;
+  optional string channel = 2;
+}
+
+message NotificationEmail {
+  enum Type {
+    TYPE_UNSET = 0;
+    TYPE_SMTP = 1;
+    TYPE_SES = 2;
+  }
+
+  optional Type type = 1;
+
+  repeated string recipients = 2;
+
+  optional string from_address = 3;
+
+  oneof config {
+    NotificationEmailSMTP smtp = 1000;
+    NotificationEmailSES ses = 1001;
+  }
+}
+
+message NotificationEmailSMTP {
+  optional string host = 1;
```

Comment:
`host` is only one that is required for smtp right?

Reply:
Just made them all required